packages/@lwc/engine/src/framework/vm.ts

```diff
@@ -51,21 +50,72 @@ export interface SlotSet {
     [key: string]: VNodes;
 }
 
+// TODO [#0]: How to get rid of the any as default generic value without passing them around through
+// the engine.
+export interface Renderer<HostNode = any, HostElement = any> {
```

Comment:
this probably should come from some other place, but I'm not sure...

Reply:
I moved this to `renderer.ts` while we discuss what we want to do with it.